deploy/6-deploy-pendle-oracle.ts

```diff
@@ -0,0 +1,59 @@
+import { ethers } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/dist/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { ADDRESSES } from "../helpers/deploymentConfig";
+
+const func: DeployFunction = async ({ getNamedAccounts, deployments, network }: HardhatRuntimeEnvironment) => {
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  const oracle = await ethers.getContract("ResilientOracle");
+  const proxyOwnerAddress = network.live ? ADDRESSES[network.name].timelock : deployer;
+
+  const { PTweETH, PTweETHMarket, PTOracle, WETH } = ADDRESSES[network.name];
```

Comment:
I would add the maturity date to the variable names of the PT token and market addresses because they will be different each time